src/components/playerOverview/playerDetails/PlayerDetailsInfoCard.tsx

```diff
@@ -23,6 +28,19 @@ export const PlayerDetailsInfoCard = React.memo(function PlayerDetailsInfoCard({
     const profileId = potentialProfileId ?? "";
     const profile = useSelector((state: AppState) => ProfileSelectors.getProfile(state, profileId));
 
+    const moxfieldProfile: MoxfieldProfile | undefined = useSelector((state: AppState) => {
+        if (state.profiles.moxfieldProfiles !== undefined && profile?.moxfieldId !== undefined) {
+            return state.profiles.moxfieldProfiles[profile.moxfieldId];
+        }
+        return undefined;
+    });
+
+    useEffect(() => {
```

Comment:
nit: can we add a comment here explaining what this useEffect is doing? makes it clearer what the intention here is (when do we hydrate?)